docs/source/dev_guide/pre_release_testing.rst

```diff
@@ -0,0 +1,59 @@
+***************************************
+Testing directions for making a release
+***************************************
+
+1. Have three shells open: one on Chrysalis, one on Compy, and one on Perlmutter. Do the following steps on each machine.
```

Comment:
It might be a good idea to have a slightly different document for testing Unified releases (i.e., using Unified RCs rather than a zppy dev environment)

Reply:
Step 3: `git checkout -b test_unified_rc12 upstream/main`
Step 6: Load the relevant Unified RC. E.g.,
```
Chrysalis: source /lcrc/soft/climate/e3sm-unified/test_e3sm_unified_1.9.0rc12_chrysalis.sh
Compy: source /share/apps/E3SM/conda_envs/test_e3sm_unified_1.9.0rc12_compy.sh
Perlmutter: source /global/common/software/e3sm/anaconda_envs/test_e3sm_unified_1.9.0rc12_pm-cpu.sh
```
Step 7: if there are any failures, make a fix and use a zppy dev environment until the fix can be incorporated into the next Unified RC.

Reply:
Include the machine name in `git checkout -b test_unified_rc12 upstream/main`. That way I can push a branch for each machine for the testing-fixes PRs that are typical in the release process (e.g., #486)